### PR TITLE
fix photo size

### PR DESCRIPTION
**Pull request by Volodia1999**

### CODE REVIEW

File: client/src/components/Post/style.scss [hunk 1]
```diff
@@ -27,3 +57,11 @@
   }
 }
 
+@media screen and (max-width: $break-md) {
```
Comment (anna-kuzii) [inline client/src/components/Post/style.scss]: pls ask Mariana how to do it better

Comment (Volodia1999) [reply client/src/components/Post/style.scss]: Fixed

File: client/src/components/Post/style.scss [hunk 2]
```diff
@@ -2,13 +2,50 @@
 
 .profile-post {
   .post-photo {
+    width: 100%;
+    position: relative;
+
+    &:hover .post-icon {
```
Comment (anna-kuzii) [inline client/src/components/Post/style.scss]: scss

Comment (Volodia1999) [reply client/src/components/Post/style.scss]: Fixed

**Pull request merged**